test/conformance/kuttl/policy-validation/cluster-policy/success/README.md

```diff
@@ -0,0 +1,11 @@
+## Description
+
+This test tries to create policies.
```

Comment:
Really? This is the best you could do? 😄

Reply:
I can imagine an even shorter description 🤷

Reply:
Codewise looks good to me🙂